ui/app/components/clients/page/overview.hbs

```diff
@@ -4,7 +4,7 @@
 ~}}
 
 <Clients::RunningTotal
-  @isSecretsSyncActivated={{@isSecretsSyncActivated}}
+  @showSecretsSync={{this.flags.secretsSyncIsActivated}}
```

Comment:
Sorry, I know this suggestion may be a pain 🙈  

Since this arg comes from `this.flags.secretsSyncIsActivated` I think keeping it`@isSecretsSyncActivated` is important here (and below) to distinguish it from the boolean that comes from the route's `showSecretsSync` method. This way a global search of `showSecretsSync` will only show the relevant args, which sets us up for future refactors/renaming (rather than having to fully rehash this logic, which we've invested a lot of energy to sort out 😅). An additional benefit, is the template will be more explicit because it will explain that the wrapped elements are reliant on activation.
```suggestion
  @isSecretsSyncActivated={{this.flags.secretsSyncIsActivated}}
```

Reply:
not a pain Claire, i absolutely welcome this kind of feedback! you're 100% right -- keeping the OG name _is_ more clear and distinguishing the arg from the arg passed down from the model is important. will update this pronto.